community/detector/cves/CVE-2021-29441/src/main/java/com/google/tsunami/plugins/detectors/cves/cve202129441/CVE202129441DetectorBootstrapModule.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.tsunami.plugins.detectors.cves.cve202129441;
+
+import com.google.tsunami.plugin.PluginBootstrapModule;
+
+/** An CVE-2021-29441 Guice module that bootstraps the {@link CVE202129441VulnDetector}. */
+public final class CVE202129441DetectorBootstrapModule extends PluginBootstrapModule {
```

Comment:
[Google Java Code Style](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case): Please use camelCase for the class name, i.e. `Cve202129441DetectorBootstrapModule`.